src/pages/Storefront/Cart.js

```diff
@@ -27,21 +27,20 @@ const CartItems = ({ checkoutData }) => {
   };
 
   return (
-    <VStack flex={1} alignItems="flex-start" spacing={8} py={16} pr={12}>
+    <VStack flex={1} alignItems="flex-start" spacing={8} pr={12}>
       <Flex w="100%" justifyContent="space-between">
-        <Heading as="h4" size="subtitle">
+        <Heading size="subtitle">
```

Comment:
same comments on heading as above — this is super minor, but it's always good to retain HTML semantics where possible